Source/f_finale.c

```diff
@@ -847,9 +847,13 @@ void F_Drawer (void)
 {
   if (using_FMI)
   {
-    if (!finalestage || !gamemapinfo->endpic[0] || (strcmp(gamemapinfo->endpic, "-") == 0))
+    if (!finalestage)
     {
       F_TextWrite();
+      if (!gamemapinfo->endpic[0])
```

Comment:
What if `gamemapinfo->endpic == '-'`?

Reply:
> What if `gamemapinfo->endpic == '-'`?

From my understanding, this doesn't happen.

Pr+ doesn't have this bug because of this code: https://github.com/coelckers/prboom-plus/blob/47beacc9f065825cf0834fb3055eee29238aa437/prboom2/src/umapinfo.cpp#L684-L689
I decided to use the original MBF code. It all became convoluted.